dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -125,6 +145,28 @@ def external_source_pipe():
 """
 
 
+_experimental_input_docstring = """Wrapper for an input passed to DALIDataset.
+Allows to pass additional options that can override some of the ones specified
+in the External Source node in the Python Pipeline object.
+Passing None indicates, that the value should be looked up in the pipeline definition.
+
+Parameters
+----------
+dataset : tf.data.Dataset
+    The dataset used as input
+layout : str, optional, default = None
+    Layout of the input. If None, the layout will be taken from the corresponding
+    External Source node in the Python Pipeline object. If it is not provided there,
+    empty layout will be used.
+batch: bool, optional, default = False
+    Batch mode of given input. If None, the batch mode will be taken from the
+    corresponding External Source node in the Python Pipeline object.
+
+    If the ``batch = False``, the input dataset is considered sample input.
```

Comment:
Shouldn't there be one backtick?

Reply:
I think we tend to use double all over the place as it's more visible. The cursive is not that distinct.